docs/_layouts/page.html

```diff
@@ -10,6 +10,9 @@
     .querySelectorAll(".markdown-body a[title]:not([href^=http])")
     .forEach((a) => {
       a.innerText = a.title;
+      if (a.href.endsWith('.md') || a.href.endsWith('.markdown')) {
```

Comment:
the same as above; maybe create an utility function to check if file is (or at least suspected to be) markdown?